tests/mongo.yml

```diff
@@ -96,9 +92,6 @@ results:
       # https://jira.mongodb.org/browse/SERVER-14832
       - mongo/jstests/core/query/distinct/distinct_array1.js
 
-      # https://github.com/FerretDB/FerretDB/issues/2236
-      - mongo/jstests/core/query/distinct/distinct1.js
-
       # https://github.com/FerretDB/FerretDB/issues/2231
```

Comment:
Hm, that one is closed. Please check

Reply:
Seems to be fixed in FerretDB/FerretDB/pull/2658. It also passes locally.

Reply:
So should we remove it from here?

Reply:
Yes I believe so, it's fixed so I removed it from fail. Same deal with auth2.js. It was fixed in FerretDB/FerretDB/pull/2726 and it passes locally.

Reply:
Oops! Let's remove them in a new PR

Reply:
Okie